pkg/tempopb/tempo.proto

```diff
@@ -219,16 +219,21 @@ message RawHistogram {
   uint64 count = 2;
 }
 
+message KeyValue {
+  string key = 1;
+  TraceQLStatic value = 2;
+}
+
 message SpanMetrics {
   repeated RawHistogram latency_histogram = 1;
-  TraceQLStatic static = 2;
+  repeated KeyValue series = 2;
```

Comment:
Maybe we also include a status code to indicate control messages like not-found?

Reply:
Or possibly on the `KeyValue`.

Reply:
When an attribute is not found it will return a traceql static with Nil type (empty value).  This works because real nils are not possible (against spec but also not persisted).